ItemLevelDownloader/Engine/worker.ts

```diff
@@ -0,0 +1,48 @@
+import { Logger } from './logger';
+export class Worker<T> {
+    private executionFunc: (item : T) => Promise<void>;
```

Comment:
knit - rename to execute to be consistent with other names or name the others with Func suffix. I prefer the former.

Reply:
done